portal-api/src/v1/validation/validateIds.js

```diff
@@ -28,25 +27,8 @@ const isValidRegex = (regex, value) => regex.test(value);
  */
 const isValidCurrencyCode = (code) => (code ? validator.isISO4217(String(code)) : false);
 
-/**
- * isNotValidCompaniesHouseNumber
- * checks if companiesHouseNumber conforms to schema
- * returns true if validation error or false if not
- * @param {String} companiesHouseNumber
- * @returns {Boolean}
- */
-const isNotValidCompaniesHouseNumber = (companiesHouseNumber) => {
```

Comment:
Can we leave the function in please with call to `isValidRegex` and make calls to this functions from tests please?